### ISSUE TITLE
Missing accessibility for blind developers using a screenreader.

### ISSUE THREAD

**Issue by robin24** [usebruno/bruno#3890]
### I have checked the following:

- [x] I use the newest version of bruno.
- [x] I've searched existing issues and found nothing related to my issue.

### Describe the bug

Hello,

I'm a blind full-stack developer and just heard about Bruno for the first time.

Unfortunately, after installing the latest version of Bruno for macOS using Homebrew, I discovered that it is not currently screenreader accessible.
As far as I can tell, this is mostly due to missing alt text on the various icons throughout the UI, making it impossible to tell, for instance, which icon needs to be clicked to create a new API request.

It would be great if accessibility could be taken into account during the ongoing development of Bruno, thanks in advance for your consideration.

### Screenshots/Live demo link

N/A

**Comment by anusree-bruno** [comment]
Hi @robin24 

Thanks for reporting this. We will look into this see how much of an effort this is. I will keep you posted.
